### PR TITLE
Class instead of function

### PR DESCRIPTION
**Pull request by Gregorek85**
Hi,

I changed function to class, added deleteItem method, added constructor for data (useful for importing csv files) and added a license.

A poza tym uczę się Git'a ;)

Pozdrawiam,
Radek Gregorczyk

### CODE REVIEW

**Comment by TeeMonk** [comment]
Hey Radek!

Thanks for submitting, I appreciate it very much! I like the `deleteItem` method you added.
With that being said - PR contains a lot of "changed" lines that were in original code. I'm still very open to add new functionalities to the code but under certain conditions:

- The pull request should contain changes related to single functionality, such as adding `deleteItem` method.
- The pull request should not show existing, unchanged lines as deleted/added, which makes the PR cluttered and hard to read. To avoid this, you should add only the part of code that is new or changed and do not touch the rest.
- If new functionality is added, documentation (README.md) should be updated as well.

Remarks regarding `deleteItem` method implementation:
- It removes the row from the sheet but leaves the corresponding item in `items` collection. If the deletion of a row is successful, the `items` should be modified as well to reflect the state present in the source sheet.
- The method name should be `deleteItem` instead of `DeleteItem` to follow naming convention for method names.

Remarks regarding `function` to `class` change:
- When the code was written the Google Apps Scrip did not support modern ECMAScript syntax. I have plans to re-write the code to utilize all syntax now avilable in V8, probably as a separate repo.

To summarize - I will close this pull request, but I would still be very happy to receive another one that follows the listed rules.  And of course - good luck with learning GIT! 👍 👍 👍
